feconf.py

```diff
@@ -782,3 +782,6 @@ def get_empty_ratings():
 
 VIEW_METHOD_ROLE = 'role'
 VIEW_METHOD_USERNAME = 'username'
+
+# Maximum number of emails admin can notify when a task fails.
+MAX_NOTIFICATION_EMAILS = 5
```

Comment:
Is this necessary? If not feel free to remove this constant. Otherwise change name to `MAX_FAILED_TRAINING_JOB_EMAILS`

Reply:
yeah you are right, 5 is anyway a good limit